README.md

```diff
@@ -40,3 +40,14 @@ The code to download it yourself is here, but you must use it yourself.
 
 This will create a `shape-up.html` file in your current directory.
 That's it.
+
+## Docker way
+
+There is a way to not deal with php at all. For that it is enough to have docker engine installed.
+
+Usage
+```
+docker build -t shapeup .
+docker run -it --rm -v $(pwd)/output:/app/output shapeup epub
```

Comment:
Can you change this from building the image to just running `thej6s/shape-up-downloader` ?
I'll setup docker hub to autobuild as soon as this PR is merged.